### ISSUE TITLE
Close buffer when deleting file

### ISSUE THREAD

**Issue by Rolv-Apneseth** [kelly-lin/ranger.nvim#11]
Hey, thanks for making this. Just a quick question, when deleting a file, is there some way to make sure the buffer for the deleted file is also closed?

**Comment by kelly-lin** [comment]
Hey @Rolv-Apneseth,

Yep, should be possible, I ran into this issue myself while using the plugin. I will add this to my todo list.

It will probably be turned on by default and the user will be prompted to confirm buffer deletion.

Thank you for the feature request, your input is appreciated.

**Comment by Rolv-Apneseth** [comment]
Awesome, thanks. Would it be possible to have a way to disable that prompt if you're going to add that? Or perhaps the option (which is enabled by default) to auto close the buffer means no prompt, and disabled means it will prompt you if you want to close?

**Comment by kelly-lin** [comment]
Yeah, it would be added in as a configuration flag for sure. However, I think it is a better user experience if the user is prompted as the default option (the opposite of what you suggested). The reasoning behind this is that it is consistent with the behaviour of neovim when you modify/delete a file from somewhere other than neovim (like the "buffer has been modified, do you want to reload the file?").

**Comment by Rolv-Apneseth** [comment]
Yeah makes sense, as long as there is an option to disable the prompt that sounds great

**Comment by Rolv-Apneseth** [comment]
If you don't have so much time to work on this, I could try have a go at implementing this, if you could just provide some guidance on how you would go about doing it (plugin noob)
